packages/position-managers/src/constants/vaults/bsc.ts

```diff
@@ -91,11 +95,12 @@ export const vaults: VaultConfig[] = [
   },
   {
     id: 15,
-    idByManager: 2,
+    idByManager: 15,
```

Comment:
@Chef-Yogi for the id by manager, I thought it's starting from 1 for each manager no?

Reply:
Yeah, ID by the manager starts from 1 for each manager, not.
Just for display

Reply:
so we don't expect that to be a unique key
![CleanShot 2024-04-10 at 15 34 35@2x](https://github.com/pancakeswap/pancake-frontend/assets/99634186/718e9277-d727-40a2-a0a2-bf4c9f364169)